javers-core/src/main/java/org/javers/common/reflection/ReflectionUtil.java

```diff
@@ -110,6 +110,41 @@ public static Object newInstance(Class clazz, ArgumentResolver resolver){
         return result;
     }
 
+    static boolean trySetMemberRelevantToGetter(JaversMethod getter, Object onObject, Object value) {
+        if (isGetter(getter.getRawMember())) {
+            String setterName = setterNameForGetterName(getter.getRawMember().getName());
+            return trySetMemberWithSetter(setterName, getter.getGenericResolvedType().getClass(), onObject, value) ||
```

Comment:
Przyjąłem, że Field/Bean Style dotyczy jedynie tego w jaki sposób określamy co jest w obrębie zainteresowania Javersa - że w Field Style określamy to na podstawie pól, a w Bean Style getterów. Zasugerowałem się tym, że napisałeś: "problemem będzie sytuacja kiedy ktoś ma niemutowalne obiekty czyli fieldy final w FIELD lub brak setterów w BEAN." Uznałem, że przy braku settera sensownym fallbackiem będzie próba ustawienia pola.
Z drugiej strony w dokumentacji Javersa jest zdanie: "BEAN style is useful for domain models compliant with Java Bean convention." Co oznacza, że przypadku Bean Style faktycznie możemy spokojnie założyć że istnieje publiczny bezargumentowy konstruktor oraz gettery i settery.